include/ensenso/ensenso_grabber.h

```diff
@@ -331,6 +331,69 @@ class PCL_EXPORTS EnsensoGrabber : public Grabber
      * stereo matching AOI in order to get valid depth data in the enlarged regions.*/
     bool setUseDisparityMapAreaOfInterest (const bool enable=false) const;
     
+    /** @brief The penalty for changes of +/- 1 disparity along an optimization path.
+       Setting a larger value for DepthChangeCost will result in smoother surfaces, but some details might get lost when setting this value too large.
+     * @param[in] A positive integer specifying the cost of disparity changes in the disparity map.
+     * @note his value must be smaller than DepthStepCost. Default Value 5
```

Comment:
You are missing the variable name (`changecost`) here in the documentation. It should be:
`* @param[in] changecost A positive integer specifying the cost...`